### ISSUE TITLE
ABB YuMi 14000 (two arm)

### ISSUE THREAD

**Issue by AndreaKondziela** [visose/Robots#60]
Dear robots-team,

have you ever used the plugin for a two arm robot? Is there a chance to get the library file for the ABB YuMi 14000?? We are currently running an interdisciplinary research project and got these robot for testing! Up to now, I only used robots for working with UR`s... Thanks in advance
Andrea

**Comment by visose** [comment]
> have you ever used the plugin for a two arm robot?

Yes, as in a robot controller connected to two robot arms using ABB's multimove. This is supported.

> Is there a chance to get the library file for the ABB YuMi 14000?? 

I've never used it. The arms are 7 axis, so a library wouldn't be enough, would need some coding. I understand there are additional RAPID functions specific for this model, I don't think it will be used as it's best with the current RAPID generated code. I think the main advantage of this model is the graphical UI and ease of programming by hand (similar to UR).
